public/edit-task.js

```diff
@@ -0,0 +1,67 @@
+const taskIDDOM = document.querySelector('.task-edit-id')
+const taskNameDOM = document.querySelector('.task-edit-name')
+const taskCompletedDOM = document.querySelector('.task-edit-completed')
+const editFormDOM = document.querySelector('.single-task-form')
+const editBtnDOM = document.querySelector('.task-edit-btn')
+const formAlertDOM = document.querySelector('.form-alert')
+const params = window.location.search
+const id = new URLSearchParams(params).get('id')
+let tempName
+
+const showTask = async () => {
```

Comment:
You must add documentation also to you frontend code, it will be helpfull for others that look on your code, and also for youself, espically when the code is messy like here. In security aspect, you should always keep in mind that this code will be seen by all application's clients, but nowdays, in real frontend frameworks, there are solutions for that (the framework swipping out the documentation before sending it to the client).